src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -2,7 +2,7 @@
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <fields>
         <fullName>Advancement_Namespace__c</fullName>
-        <defaultValue>&quot;adv1&quot;</defaultValue>
```

Comment:
The change itself is fine, however any new installs of NPSP since this field was first implemented would have a value in the field of "adv1". Is that an issue, or is it assumed that existing NPSP customers (even ones that were spun up since the field was introduced) would not likely be GEM users as well?

Reply:
@bdvorachek Just read through the quip doc and you're already considering that the field value will be "adv1" for current NPSP customers. Approving. Is this going to QA?

Reply:
We know who all has the GEM pilot installed, this shouldn't break any existing installs since they'll have the "adv1" namespaced GEM installed still. New pilot installs that happen in the window between this being released and GEM being updated are the only risk here, and we should know about those too.

I'm not sure what the QA plan for this would be, since we need to also make a change in GEM to support this.

Reply:
@annejackson

Reply:
@force2b - for this change, are there certain things you'd like to see tested in a Cumulus scratch org? We can do some testing in GEM with the beta_dep flow after merging this. Please let me know if you have some areas of concern (either testing functionality in the UI like making sure logging a donation works fine, or checking something in custom settings, or if it's along the lines of just doing some tests via dev console)? CC: @bdvorachek

Reply:
I was more curious than anything @annejackson. I wasn't sure if this needed to go through some tests in an org that had a previous value in that field (though I suppose that wouldn't be possible for unmanaged testing). I guess if you really wanted to test the variations that  @bdvorachek mentioned it could be done post-merge using a beta package.